### PR TITLE
rbx_reflection: has_superclass

### PR DESCRIPTION
**Pull request by krakow10**
Hi, I keep writing this function and thought it could be provided here.  Open to suggestions for the function name and signature!

### CODE REVIEW

File: rbx_reflection/src/database.rs [hunk 1]
```diff
@@ -86,6 +86,17 @@ impl<'a> ReflectionDatabase<'a> {
         }
     }
 
+    /// This mimics the behavior of the Roblox Lua Instance:IsA(ClassName) method.
+    /// Returns whether superclass_descriptor is a superclass of descriptor.
```
Comment (Dekkonot) [inline rbx_reflection/src/database.rs:90]:
```suggestion
    /// This mimics the behavior of the Roblox method `Instance:IsA(ClassName)` method.
    /// Returns whether `superclass_descriptor` is a superclass of `descriptor`.
```

**Review by Dekkonot — COMMENTED** [review]
I'm open to including this. I think `class_is_a` is a bit weird, but the type signature is fine.

This needs an entry in `rbx_reflection`'s changelog though.

**Comment by krakow10** [comment]
> I'm open to including this. I think `class_is_a` is a bit weird, but the type signature is fine.

Do you have an idea of what to change it to?  The Roblox function is also poorly named.

**Comment by Dekkonot** [comment]
I think `has_superclass` is probably good. If it has a doc comment explaining what it does, I think the name is intuitive enough that it won't confuse anyone.

**Review by Dekkonot — APPROVED** [review]

**Pull request merged**